database/migrations/2020_01_30_155905_add_contentful_campaign_i_d_to__campaign_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddContentfulCampaignIDToCampaignTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->string('contentful_campaign_id')->nullable->comment('Add the related contentful campaign id when applicable.');
```

Comment:
Let's add this column after the `id` column so it doesn't show up all the way at the end after the timestamps and everything